apps/dash-web-trader/app.py

```diff
@@ -0,0 +1,1474 @@
+                                ],
+                                value="colored_bar_trace",
+                            )
+                        ],
+                    ),
+                ],
+            ),
+            # Chart Top Bar
+            html.Div(
+                className="row chart-top-bar",
+                children=[
+                    html.Span(
+                        id=pair + "menu_button",
+                        className="inline-block chart-title",
+                        children=f"{pair} ☰",
+                        n_clicks=0,
+                    ),
+                    # Dropdown and close button float right
+                    html.Div(
+                        className="graph-top-right inline-block",
+                        children=[
+                            html.Div(
+                                className="inline-block",
+                                children=[
+                                    dcc.Dropdown(
+                                        className="dropdown-period",
+                                        id=pair + "dropdown_period",
+                                        options=[
+                                            {"label": "5 min", "value": "5Min"},
+                                            {"label": "15 min", "value": "15Min"},
+                                            {"label": "30 min", "value": "30Min"},
+                                        ],
+                                        value="15Min",
+                                        clearable=False,
+                                    )
+                                ],
+                            ),
+                            html.Span(
+                                id=pair + "close",
+                                className="chart-close inline-block float-right",
+                                children="×",
+                                n_clicks=0,
+                            ),
+                        ],
+                    ),
+                ],
+            ),
+            # Graph div
+            html.Div(
+                dcc.Graph(
+                    id=pair + "chart",
+                    className="chart-graph",
+                    config={"displayModeBar": False, "scrollZoom": True},
+                )
+            ),
+        ],
+    )
+
+
+# returns modal Buy/Sell
+def modal(pair):
+    return html.Div(
+        id=pair + "modal",
+        className="modal",
+        style={"display": "none"},
+        children=[
+            html.Div(
+                className="modal-content",
+                children=[
+                    html.Span(
+                        id=pair + "closeModal", className="modal-close", children="×"
+                    ),
+                    html.P(id="modal" + pair, children=pair),
+                    # row div with two div
+                    html.Div(
+                        className="row",
```

Comment:
The margin here is really thin, would it look better with a little spacing? 
<img width="573" alt="Screen Shot 2019-07-04 at 1 04 43 PM" src="https://user-images.githubusercontent.com/23962118/60681189-54e53c80-9e5c-11e9-93c0-f42432d76b57.png">